taxi_service/settings.py

```diff
@@ -20,9 +21,8 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
+SECRET_KEY = \
     "django-insecure-8ovil3xu6=eaoqd#-#&ricv159p0pypoh5_lgm*)-dfcjqe=yc"
-)
```

Comment:
Why you change this?
Better to use `()` then `\`

Reply:
Flake8 said that it is too much of symbols